qiskit/pulse/utils.py

```diff
@@ -125,3 +130,24 @@ def deprecated_functionality(func):
         stacklevel=2,
         since="0.22.0",
     )(func)
+
+
+def get_qubit_channels(backend: BackendV2, qubit: int):
```

Comment:
Sorry my previous comment was wrong. This is also API change (i.e. new feature) and we should wait for another release. To make this PR a part of patch release, you need to embed this logic in the `qubit_channels` to avoid adding new public API.

Reply:
Fixed at https://github.com/Qiskit/qiskit-terra/pull/10150/commits/9b2b80d15c1ff4f667d830bda6350d478f15c9b0.